docs/UserGuide.adoc

```diff
@@ -404,7 +404,7 @@ The calendar has a day view, week view and month view to help you view all your
 
 ==== Displaying the calendar : `calendar` `since v1.3`
 
-Displays the Calendar at centre of window in specified viewing format. +
+Displays the Calendar at center of interface in specified viewing format. +
```

Comment:
Maybe this can be changed to `displays the Calendar in the CenterPanel` since we have defined `CenterPanel` in the glossary?

Reply:
Alright, Thanks!